### PR TITLE
Adds the Heckler and Koch mp5sd submachinegun

### PR DESCRIPTION
**Pull request by hyper2snyper**
```SUMMARY: Content "Adds the h&k mp5sd sub machine gun."```
https://www.heckler-koch.com/en/products/military/submachine-guns/mp5/mp5sd/overview.html
The official gun review. A very rare gun, so I made it only found in military bases with a spawn weight of 75.
It is silenced and easier to handle than the mp5, however beside that it is nearly identical.

### CODE REVIEW

**Comment by OrenAudeles** [comment]
Summary should be in the following format
```SUMMARY: <category> "<description>"```
so, for this one: 
```SUMMARY: Content "Adds the h&k mp5sd sub machine gun"```

**Comment by hyper2snyper** [comment]
What does the failed build mean? Did I do something wrong?

**Comment by tenmillimaster** [comment]
Isn't a spawn weight of 75 pretty common? I'd say the spawn weight should be much rarer on these. They're not exactly common.

**Comment by hyper2snyper** [comment]
it is in the military spawn group, so itll only spawn in military sites. should i up it to 125?

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -157,6 +157,19 @@
       [ "46", [ "hk46mag", "hk46bigmag" ] ],
       [ "57", [ "fn57mag" ] ]
     ]
+  },  
+  {
+    "id": "hk_mp5sd",
+    "copy-from": "hk_mp5",
+    "type": "GUN",
+    "reload_noise_volume": 10,
```
Comment (ZhilkinSerg) [inline data/json/items/gun/9mm.json]:
```suggestion
```

image

**Pull request merged**